<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountDialog</class>
 <widget class="QDialog" name="CountDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>291</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Count Actors</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Count</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>operator</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QComboBox" name="operatorBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>attribute</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <widget class="QComboBox" name="attributeBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>What freatiure of the agent to fileter by</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>value</string>
       </property>
       <widget class="QDoubleSpinBox" name="valueSpinBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>62</width>
          <height>24</height>
         </rect>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_7">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="title">
        <string>days</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QSpinBox" name="dayspinBox_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="dayspinBox_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CountDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
